manifests/poc-data-cluster/prod/spark/iot-rewards-silver.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: iot-poc-silver-query
+  namespace: spark
+data:
+  query.sql: |
+    SELECT 
+      start_period,
+      end_period,
+      gateway_reward
+      FROM iot_reward_share
```

Comment:
If we're only taking gateway_reward, recommend (1) exploding this out by selecting the individual values and (2) filtering where gateway_reward is not null